ShopifySharp/Entities/ProductListingVariant.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace ShopifySharp
+{
+    public class ProductListingVariant
+    {
+        /// <summary>
+        /// The object's unique id.
+        /// </summary>
+        [JsonProperty("id")]
+        public long Id { get; set; }
+        
+        /// <summary>
+        /// Available current product variant
+        /// </summary>
+        [JsonProperty("available")]
+        public bool Available { get; set; }
```

Comment:
We should make this a nullable bool for consistency with the rest of the package, even if the Shopify docs say it won't be null:

```
public bool? Available { get; set; }
```